### PR TITLE
FRONTEND: migrate MUI v4 -> MUI v5

### PR DESCRIPTION
**Pull request by epicoding95**
This PR should contain all of the necessary code for WUI to expose MUI v5 styles/components. The only other thing needed will be the actual publishing of this package to the NPM registry.

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -89,4 +89,8 @@ const theme = createTheme({
 //   set because they are used in the method.
 theme.typography = addTypography(theme);
 
-export default theme;
+function useTheme() {
+  return theme;
+}
```
Comment (markrawls) [inline src/theme.js:94]: Something something "use arrow functions instead of named functions"

**Review by markrawls — CHANGES_REQUESTED** [review]
LGTM otherwise!

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
-export { default as theme } from './theme';
+export { makeStyles, useStyles } from './theme';
```
Comment (markrawls) [inline src/index.js:1]: I think we probably still want to export the theme object as well, right? In the current form of this PR it's inaccessible and we still need to pass it to the `ThemeProvider`.

**Review by markrawls — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (epicoding95) [reply src/index.js:1]: Unless i'm missing something, there is no `ThemeProvider` in wui? If we do need to access the theme, that's exactly what useStyles provides

Comment (markrawls) [reply src/index.js:1]: WUI does not expose a `ThemeProvider` but MUI does, and the app using WUI needs to put the WUI theme in there.

Comment (epicoding95) [reply src/index.js:1]: ahh right, fixing now

**Review by markrawls — APPROVED** [review]
Looks good to me!

**Pull request merged**